### ISSUE TITLE
 The first letter is not kept in chat

### ISSUE THREAD

**Issue by tigernero79** [nextcloud/talk-android#190]
With Nextcloud Talk, when I start a conversation in public chat via link sharing, when I start writing on smartphone in the chat, the first letter is not kept, if I write "test" exits written "est" I have to click t twice. this error does it only from smartphone browsers on pc works well

**Comment by mario** [comment]
I am not sure I understand? You are talking about a mobile browser, and not a Nextcloud Talk Android app?

**Comment by tigernero79** [comment]
Yes, I'm talking about a mobile browser, if I click on the generated link of the public chat, I'll log in, but when I start writing a word, the first character is not retained if I click it twice. example "test" becomes "east" without "t" if I click the initial twice then the test t remains
